apps/academy-wep/components/FeaturedLearningPaths.tsx

```
@@ -0,0 +1,63 @@
+
+  useEffect(() => {
+    setTimeout(() => {
+      setLearningPathsCount(6);
+    }, 2000);
+  }, []);
+
+  return (
+    <div className="bg-gray-50 rounded-lg shadow-lg p-4 md:p-6 mx-auto">
+      <div className="flex items-center mb-4">
+        <span className="bg-green-100 text-green-600 text-sm font-semibold px-3 py-1 rounded-full flex items-center">
+          <i className="fas fa-bolt mr-2"></i> Featured Learning Paths
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)